test/test_latex.py

```diff
@@ -1,12 +1,13 @@
 """
 Tests for uqcsbot.scripts.cat
 """
```

Comment:
can you move this to be a function doc instead like the other tests? (and update it from being for !cat)